ui/webui/resources/sources.gni

```diff
@@ -13,16 +13,22 @@ brave_resources_extra_grdps = []
 brave_resources_extra_grdps_deps = []
 
 # ios does not need any static resources
-if (!is_ios) {
+if (is_ios) {
+  brave_resources_extra_grdps += [
+    "$brave_resources_extra_grdps_path/brave_static_resources.grdp",
```

Comment:
I think this should move to https://github.com/brave/brave-core/pull/26986/files#diff-afa0b278b965b62d1347bc405d28d7ebc70ad2e1f8c14394b4019238bde0a9ebR13 ? Also the comment here seems wrong now

Reply:
and of course https://github.com/brave/brave-core/pull/26986/files#diff-afa0b278b965b62d1347bc405d28d7ebc70ad2e1f8c14394b4019238bde0a9ebR24 should be removed

Reply:
Fixed both.